lang/sql/sql.talon

```diff
@@ -0,0 +1,38 @@
+mode: command
+and mode: user.sql
+mode: command
+and mode: user.auto_lang
+and code.language: sql
+-
+tag(): user.code_generic
+tag(): user.code_operators
+
+# Some commands have spaces around them to avoid having to continually say 'space'.
+# The ones that don't have an initial space are expected to be at the start of a new line.
+# This could probably be done in a better way.
+select: "SELECT "
+star: "* "
+distinct: "DISTINCT "
+from: "FROM "
+case: "CASE"
+when: "WHEN "
+then: "THEN "
+else: "ELSE "
+end: "END"
+as: " AS "
+where: "WHERE "
+between: " BETWEEN "
+and: "AND "
+or: "OR "
+is: "IS "
+not: "NOT "
+in: "IN "
```

Comment:
Is `NOT` ever used without `IN` or `NULL`? Same as `IS NULL` and `IS NOT NULL` above.

Here's how I implement `IN`/`NOT IN` as operators: https://github.com/nriley/knausj_talon/blob/nriley/lang/sql/sql.py#L29